lib/index.js

```diff
@@ -567,18 +567,32 @@ Assertion.prototype.expects = function(){
 };
 
 /**
- * Assert that the integration errors with optional `msg`.
+ * Assert that the integration errors with optional error `msg` or `status`.
  *
- * @param {String} msg
+ * @param {String} [msg]
+ * @param {Number} [status]
  * @param {Function} fn
  * @api public
  */
 
-Assertion.prototype.error = function(msg, fn){
-  if (1 == arguments.length) fn = msg, msg = null;
+Assertion.prototype.error = function(){
+  var args = [].slice.call(arguments)
+  var fn = args.pop()
```

Comment:
Did the linter not bother that you're not using semicolons? Impressive how inconsistent our formatting rules are :P